textdb/textdb-sandbox/team3datafile.txt

```diff
@@ -0,0 +1,4 @@
+networkx 1.1.1
```

Comment:
@laisycs .. Sorry for being late. You can include the following chanegs in the next pull request..
1. Include your data files in src/test/resources/team3 or src/main/resources/team3 (depending on whether the data files are used by the test cases or the main classes) folder. 
2. Make src/test/resources as source folder.